README.md

```diff
@@ -71,7 +71,7 @@ The following prefixes are currently available to use:
 
 - `ri` prefix holds [**Remix Icon** 2.5.0 with its 2271 icons.](https://github.com/Remix-Design/RemixIcon)
 
-- `msc` prefix holds Microsoft's [**Codicons** 0.0.35 with its 446 icons.](https://github.com/microsoft/vscode-codicons)
```

Comment:
How is that the version is the same but the number of icons is different? Shouldn't the version number be greater than `0.0.35`?

Reply:
I think this is caused due to the change from the `.csv` to the `.css` file as source to generate the icons charmap file. As mentioned at https://github.com/spyder-ide/qtawesome/pull/241#issue-1799283041 the `.csv` misses icons definitions available from the `.css` file.